web/src/lib/stores/assets.store.ts

```diff
@@ -175,7 +175,7 @@ export class AssetStore {
     for (const bucket of this.buckets) {
       if (height < viewport.height) {
         height += bucket.bucketHeight;
-        this.loadBucket(bucket.bucketDate, BucketPosition.Visible);
+        await this.loadBucket(bucket.bucketDate, BucketPosition.Visible);
```

Comment:
Will cause buckets to load sequentially and might slow down initial loading a bit